worker/deps/libwebrtc/libwebrtc/modules/congestion_controller/goog_cc/alr_detector.h

```diff
@@ -23,6 +24,19 @@
 
 namespace webrtc {
 
+class RtcEventLog;
+
+struct AlrDetectorConfig {
+  // Sent traffic ratio as a function of network capacity used to determine
+  // application-limited region. ALR region start when bandwidth usage drops
+  // below kAlrStartUsageRatio and ends when it raises above
+  // kAlrEndUsageRatio. NOTE: This is intentionally conservative at the moment
+  // until BW adjustments of application limited region is fine tuned.
+  double bandwidth_usage_ratio = 0.65;
```

Comment:
might bw worth to add a defined scalar value for the drop as well to trigger the detector.
for example if the throughput is 90 and it drops to 65, it is a congestion collapse, but it will not trigger the detector, because the drop is not enough, but in reality its a significant drop